__tests__/project.test.js

```diff
@@ -166,3 +168,38 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("accepts a body with username and body responds with the posted comment", () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send({ username: "butter_bridge", body: "I really like Wing Chun" })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.article).toHaveProperty("comment_id", 19);
+        expect(body.article).toHaveProperty("body", "I really like Wing Chun");
+        expect(body.article).toHaveProperty("article_id", 2);
+        expect(body.article).toHaveProperty("author", "butter_bridge");
+        expect(body.article).toHaveProperty("votes", 0);
+        expect(body.article).toHaveProperty("created_at");
+      });
+  });
```

Comment:
This is a really good test, but this should not be on a key of article. As your test description suggests this should be the new comment, so I would recommend renaming this to simply be comment